### PR TITLE
fix: hero section image grid

### PR DESCRIPTION
**Pull request by Sanchitbajaj02**
## Related Issue

As mentioned in issue #1032, the home page image grid is not looking good on mobile screens. The issue I detected is related to the max-width issue and excessive use of absolute properties. 

## Description

To solve this issue, I first tried to remove all the dependencies related to absolute classes and use a grid to make 2 columns and align the content in them but to remove the existing code and rewrite it for all different screen sizes was kind of a hassle and disrespect to all the contributors who worked hard to create that section.
So, to fix that, I optimize the existing absolute classes for different screen sizes and gave a global max width class to set the content's max width.

### Before
image

### After
image

### CODE REVIEW

File: src/pages/HomePage.jsx [hunk 1]
```diff
@@ -10,77 +10,34 @@ import 'react-toastify/dist/ReactToastify.css';
...
       position: 'top-right',
-      autoClose: 5000,
+      autoClose: 3000,
       hideProgressBar: false,
       closeOnClick: true,
       pauseOnHover: true,
       draggable: false,
       progress: undefined,
-      theme: 'light',
+      theme: theme.mode,
```
Comment (priyankarpal) [inline src/pages/HomePage.jsx:22]: revert the change. we added only light bg  
```suggestion
      theme: 'light,
```

**Review by priyankarpal — CHANGES_REQUESTED** [review]
also, please check the right side images problem

image

File: src/pages/HomePage.jsx (hunk 1 above)
Comment (Sanchitbajaj02) [reply src/pages/HomePage.jsx:22]: sure, I will switch that back. I thought it looks cool

**Comment by Sanchitbajaj02** [comment]
> also, please check the right side images problem
> 
> image

can you tell on which screen size, does this issue occurred?

**Comment by priyankarpal** [comment]
> > also, please check the right side images problem
> > 
> > image
> 
> can you tell on which screen size, does this issue occurred?

1920 x 1080

**Comment by priyankarpal** [comment]
> > > also, please check the right side images problem
> > > image
> > 
> > 
> > can you tell on which screen size, does this issue occurred?
> 
> 1920 x 1080

still same problem

**Comment by Sanchitbajaj02** [comment]
I have reverted back the max-width part. Can we come on call if this still gives the same issue?

To reproduce this:-

> image

I reduced the max-width manually. I couldn't get this same issue on the screens I have tested

**Comment by priyankarpal** [comment]
> I have reverted back the max-width part. Can we come on call if this still gives the same issue?
> 
> To reproduce this:-
> 
> > image
> 
> I reduced the max-width manually. I couldn't get this same issue on the screens I have tested

yeah

**Comment by Sanchitbajaj02** [comment]
> > I have reverted back the max-width part. Can we come on call if this still gives the same issue?
> > To reproduce this:-
> > > image
> > 
> > 
> > I reduced the max-width manually. I couldn't get this same issue on the screens I have tested
> 
> yeah

issue still occurring?

**Comment by priyankarpal** [comment]
> > > I have reverted back the max-width part. Can we come on call if this still gives the same issue?
> > > To reproduce this:-
> > > > image
> > > 
> > > 
> > > I reduced the max-width manually. I couldn't get this same issue on the screens I have tested
> > 
> > 
> > yeah
> 
> issue still occurring?

nope but mobile view 

> > > I have reverted back the max-width part. Can we come on call if this still gives the same issue?
> > > To reproduce this:-
> > > > image
> > > 
> > > 
> > > I reduced the max-width manually. I couldn't get this same issue on the screens I have tested
> > 
> > 
> > yeah
> 
> issue still occurring?

## Mobile view

image


## Desktop view
Screenshot from 2023-06-04 16-11-23

**Comment by priyankarpal** [comment]
Have you reverted the changes? @Sanchitbajaj02

**Comment by Sanchitbajaj02** [comment]
> Have you reverted the changes? @Sanchitbajaj02

I reverted the max-width problem that was occurring. I fix the card grid for 350 screen size and above

**Review by priyankarpal — APPROVED** [review]
